nf_core/modules/lint/main_nf.py

```diff
@@ -235,54 +235,8 @@ def check_process_section(self, lines, fix_version, progress_bar):
         self.failed.append(("process_capitals", "Process name is not in capital letters", self.main_nf))
 
     # Check that process labels are correct
```

Comment:
You still need to call your new function here right? :thinking:

Reply:
😂 Yes I completely missed that